docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -15,48 +15,241 @@ related:
     link: docs/scos/user/back-office-user-guides/page.version/merchandising/discount/managing-discounts.html
 ---
 
-This guide describes how to create a cart rule.
+This guide describes how to create a *cart rule*.
 
 A cart rule is a discount that is applied automatically when all attached discount conditions are fulfilled and if the cart rule is active. Unlike a voucher code, it does not require any input from the customer.
 
 ## Prerequisites
 
-To start working with discounts, go to **Merchandising** > **Discount**.
+To start working with discounts, go to **Merchandising&nbsp;<span aria-label="and then">></span> Discount**.
 
-## Creating a cart rule discount
+Review the reference information before you start, or just look up the necessary information as you go through the process.
+
+## Creating cart rule discounts
 
 To create a cart rule discount:
 1. On the **Discount** page, in the top right corner, click **Create new discount**.
-2. On the **Create Discount** page, on the **General information** tab, do the following:
-    1. In **STORE RELATION**, check the stores you wish the discount to be active in.
-    2. From the **DISCOUNT TYPE** drop-down list, select a cart rule.
-    3. In the **NAME** field, specify the name for the discount.
-    4. _Optional_: in the **DESCRIPTION** field, enter the description of the discount. The description is displayed in the cart, in the **Promotional products** section.
-    5. _Optional_: in the **PRIORITY** field, enter an integer value from `1` to `9999` for the discount priority. For reference information, in the **Create new discount** reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
-    6. Specify if the discount is exclusive. For reference information, in the **Create new discount** reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
-    7. Specify the validity interval (lifetime) of the discount.
-* Click **Next** or select the **Discount calculation** tab to proceed.
+2. On the **Create new discount** page, in the **General Information** tab, do the following:
+   1. In **STORE RELATION**, select stores you wish the discount to be active in.
+   2. For **DISCOUNT TYPE**, select **Cart rule**.
+   3. In the **NAME** field, specify the name for the discount.
+   4. Optional: in the **DESCRIPTION** field, enter a summary that explains the discount.
```

Comment:
The first letter after `Optional:` should be capitalized. See [Optional steps](https://developers.google.com/style/procedures?hl=en#optional-steps).

Reply:
Apply this in the other steps too.